EIPS/eip-7121.md

```diff
@@ -0,0 +1,81 @@
+---
+eip: 7121
+title: SoulBounds Interface
+description: Interface for representing and managing soulbound assets based on the ERC-1155
+author: Leonid Shaydenko (@lssleo)
+discussions-to: https://ethereum-magicians.org/t/erc-7121-soulbounds-interface/14549
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-01
+requires: 165, 1155
+---
+
+## Abstract
+
+The SoulBounds EIP proposes a standard interface, called SoulBounds, that extends the `ERC-1155` token standard to support the representation and management of soulbound assets on the Ethereum network. Soulbound assets are unique digital items that are permanently bound to a specific user or address, preventing transferability to other addresses.
+
+## Motivation
+
+The concept of soulbound assets is commonly used in gaming and non-fungible token (NFT) applications, where certain items or collectibles are intended to be non-transferable. Currently, there is no standardized way to represent and manage soulbound assets on the Ethereum network. By introducing the SoulBounds interface, I aim to establish a common standard for developers to create, manage, and interact with soulbound assets in a consistent and interoperable manner.
```

Comment:
Try to avoid first person.

```suggestion
The concept of soulbound assets is commonly used in gaming and non-fungible token (NFT) applications, where certain items or collectibles are intended to be non-transferable. Currently, there is no standardized way to represent and manage soulbound assets on the Ethereum network. The SoulBounds interface aims to establish a common standard for developers to create, manage, and interact with soulbound assets in a consistent and interoperable manner.
```